pkgs/applications/office/fantasia-archive/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, fetchzip
+, appimageTools
+, makeDesktopItem
+}:
+
+let
+  pname = "fantasia-archive";
+  version = "0.1.10";
+
+  archive = fetchzip {
+    url = "https://github.com/Elvanos/fantasia-archive/releases/download/v${version}/Fantasia.Archive-${version}-linux-appImage.zip";
+    hash = "sha256-Ls+7wmj2qFoppiHJwt9Uan+9irFvzHxEWjIWrA9Nw2w=";
+  };
+  src = "${archive}/Fantasia.Archive-${version}-linux-appImage.AppImage";
+  appimageContents = appimageTools.extractType2 {
+    inherit pname src version;
+  };
+
+  description = "100% free, powerful & feature-rich offline worldbuilding tool that runs on your computer!";
+
+  desktopItem = makeDesktopItem {
+    name = pname;
+    exec = "sh -c \"${pname} || ${pname} --disable-gpu-sandbox\"";
```

Comment:
As mentioned in the Nix Discord server, this flag was added because the program refuses to start with the GPU sandbox enabled.

Reply:
Right. Is it alright to leave this in the desktop item, then?

Reply:
Yep, but we should try to figure out why we need to add that flag.